CHANGES.md

```diff
@@ -231,6 +231,11 @@ OpenSSL 3.0
 
    *Richard Levitte*
 
+ * Renamed the `-startdate` option of the `x509` command to `-notBefore` and
```

Comment:
Please drop this change. It is useless. The options were there since ages. And you're making this inconsistent with ca command.

Reply:
> Please drop this change. It is useless. The options were there since ages. 
And you're making this inconsistent with ca command.

This change was suggested by @InfoHunter who has been with the project even longer than this option.
See the discussion above: https://github.com/openssl/openssl/pull/13711#pullrequestreview-556251087
To me, either variant is fine; does anyone else have a strong opinion on it?

Reply:
As I understand the original @InfoHunter comment it was about the help text, not about the names of the options.

Reply:
Argh, indeed I must have entirely misunderstood his comment!
Followed your interpretation.